src/datasets/commands/env.py

```diff
@@ -1,54 +1,29 @@
 import platform
+import sys
 from argparse import ArgumentParser
 
 from datasets import __version__ as version
-from datasets import config
-from datasets.commands import BaseTransformersCLICommand
+from datasets.commands import BaseDatasetsCLICommand
 
 
 def info_command_factory(_):
     return EnvironmentCommand()
 
 
-class EnvironmentCommand(BaseTransformersCLICommand):
+class EnvironmentCommand(BaseDatasetsCLICommand):
     @staticmethod
     def register_subcommand(parser: ArgumentParser):
-        download_parser = parser.add_parser("env")
+        download_parser = parser.add_parser("env", help="Print relevant system environment info.")
         download_parser.set_defaults(func=info_command_factory)
 
     def run(self):
-        pt_version = "not installed"
-        pt_cuda_available = "NA"
-        if config.TORCH_AVAILABLE:
-            import torch
-
-            pt_version = torch.__version__
-            pt_cuda_available = torch.cuda.is_available()
-
-        tf_version = "not installed"
-        tf_cuda_available = "NA"
-        if config.TF_AVAILABLE:
-            import tensorflow as tf
-
-            tf_version = tf.__version__
-            try:
-                # deprecated in v2.1
-                tf_cuda_available = tf.test.is_gpu_available()
-            except AttributeError:
-                # returns list of devices, convert to bool
-                tf_cuda_available = bool(tf.config.list_physical_devices("GPU"))
-
         info = {
             "`datasets` version": version,
-            "Platform": platform.platform(),
+            "Platform": sys.version,
```

Comment:
We actually need the `platform.platform()` here. This is useful to know if users are on windows or not for example.